Sources/CacheAdvance/FileHandleExtensions.swift

```diff
@@ -21,12 +21,57 @@ extension FileHandle {
 
     // MARK: Internal
 
+    /// A method to read data from a file handle that is safe to call in Swift from any operation system version.
+    func readDataUp(toLength length: Int) throws -> Data {
+        if #available(iOS 13.0, tvOS 13.0, watchOS 6.0, OSX 15.0, *) {
+            return try __readDataUp(toLength: length)
+        } else {
+            return try ObjectiveC.unsafe { readData(ofLength: length) }
+        }
+    }
+
+    /// A method to write data to a file handle that is safe to call in Swift from any operation system version.
+    func write(data: Data) throws {
+        if #available(iOS 13.0, tvOS 13.0, watchOS 6.0, OSX 15.0, *) {
+            return try __write(data, error: ())
+        } else {
+            return try ObjectiveC.unsafe { write(data) }
+        }
+    }
+
+    /// A method to seek on a file handle that is safe to call in Swift from any operation system version.
+    func seek(to offset: UInt64) throws {
```

Comment:
I don't love this method name. I'm open to suggestions.

Reply:
this should probably be `seek(toOffset offset` since UInt64 contains [weak type information](https://swift.org/documentation/api-design-guidelines/#weak-type-information), but besides that I think seek is fine.

Reply:
@fdiaz I agree! But iOS 13's API already took that name 😂